env.example

```diff
@@ -223,3 +223,5 @@ JIBRI_XMPP_PASSWORD=
 
 # Jitsi image version (useful for local development)
 #JITSI_IMAGE_VERSION=latest
+
+#P2P_STUN_SERVERS=meet-jit-si-turnrelay.jitsi.net:443,stun.l.google.com:3478
```

Comment:
Please drop this. I want to keep the example file minimal and the options documented in the handbook.

Reply:
Done